src/LightPanels/LightPanels.less

```diff
@@ -95,6 +95,7 @@
 
 		> .offscreen {
 			visibility: hidden;
+			opacity: 0;
```

Comment:
Hi @yeram, no problem! If we do not set `visibility: hidden`, the panel will still respond to events and affect tab ordering.